package.json

```diff
@@ -27,10 +27,17 @@
     "url": "https://github.com/strongloop/loopback-soap/issues"
   },
   "devDependencies": {
+    "chai": "^3.2.0",
     "eslint": "^3.12.2",
-    "eslint-config-loopback": "^7.0.0"
+    "eslint-config-loopback": "^8.0.0",
+    "loopback": "^3.0.0",
```

Comment:
Normally I'd ask this to be removed until then, but you've already put up with enough of my demands I'll let this one slide :-P